spec/defines/seboolean_spec.rb

```diff
@@ -0,0 +1,81 @@
+require 'spec_helper'
+describe 'types::selboolean' do
```

Comment:
We want to ensure that there is a test for each parameter showing what happens with a valid value and what happens with an invalid type or invalid value if there is one. Such as when using validate_re() as you are doing. Since you are doing that, we would test with value being 'on' and 'off'.